dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/DiscretePI.mo

```diff
@@ -19,24 +19,32 @@ block DiscretePI "Proportional integrator with discrete input"
 
   parameter Real Gain "Control gain";
   parameter Types.Time tIntegral "Time integration constant";
+  parameter Real Y0 "Start value of the PI output ";
```

Comment:
As it is a start value, this should be juste before the equation section. Also you can remove the space before ". Thanks